language/move-prover/spec-lang/src/ast.rs

```diff
@@ -59,7 +60,9 @@ pub enum InvariantKind {
 
 #[derive(Debug)]
 pub struct Invariant {
+    pub loc: Loc,
     pub kind: InvariantKind,
+    pub target: Option<(ModuleId, SpecVarId)>,
```

Comment:
Comment for what presence/absence of this means?

Reply:
Done (when I push)